Makefile.tt

```diff
@@ -0,0 +1,57 @@
+include .env
+
+.PHONY: db/migrate db/rollback db/wait-for-postgres db/seed dev env/setup env/teardown test test/docker codebase codebase/fix
+
+db/migrate:
+	make db/wait-for-postgres
+	rails db:migrate
+
+db/rollback:
+	make db/wait-for-postgres
+	rails db:rollback
+
+db/wait-for-postgres:
+	$(shell DATABASE_URL=$(DATABASE_URL) ./bin/wait-for-postgres.sh)
```

Comment:
After quite some months using the `make db/*` commands, I realized they might be misleading.
Because by abstracting the db commands, we put the developer away from using properly the rails/rake DB commands.
For example, I often want to `rake db:migrate:redo`, and with the current MakeFile, it is not possible. 

Also I do not think we want to include in the make file all [the possible commands](https://gist.github.com/sergii/6105745), so what about removing these 3 commands? 🤔

Reply:
absolutely, deleted all db related make commands in favor of using rake tasks, including `db/seeds` as well
https://github.com/nimblehq/rails-templates/pull/359/commits/c4aeed2b779d192b298409e399e41a0d61de34d3 ✅